src/cqueue.rs

```diff
@@ -184,6 +184,9 @@ impl ExactSizeIterator for CompletionQueue<'_> {
 impl Entry {
     /// The operation-specific result code. For example, for a [`Read`](crate::opcode::Read)
     /// operation this is equivalent to the return value of the `read(2)` system call.
+    /// Except, in an error case many system calls would return `-1` and set errno.
```

Comment:
I believe that the meaning of `result` is not universal, and the meaning of `result` of each opcode may be different. We should mention this here and describe the meaning of `result` in each opcode document.